packages/clerk-js/src/ui/elements/LegalConsentCheckbox.tsx

```diff
@@ -71,7 +68,7 @@ export const LegalCheckbox = (
       <Flex justify='center'>
         <Field.CheckboxIndicator
           elementDescriptor={descriptors.formFieldCheckboxInput}
-          elementId={descriptors.formFieldInput.setId('__experimental_legalAccepted')}
+          elementId={descriptors.formFieldInput.setId('legalAccepted')}
         />
         <FormLabel
```

Comment:
Wonder if the checkbox label should dim when the form is submitting similar to the other input labels. It currently stays at full opacity but the checkbox input dims.


https://github.com/user-attachments/assets/333b092f-c206-493c-89a5-bcca0450305b

Reply:
Fixed!